src/server/qgsbufferserverresponse.h

```diff
@@ -0,0 +1,125 @@
+/***************************************************************************
+                          qgsfcgiserverresponse.h
+
+  Define response wrapper for storing responsea in buffer
+  -------------------
+  begin                : 2017-01-03
+  copyright            : (C) 2017 by David Marteau
+  email                : david dot marteau at 3liz dot com
+ ***************************************************************************/
+
+/***************************************************************************
+ *                                                                         *
+ *   This program is free software; you can redistribute it and/or modify  *
+ *   it under the terms of the GNU General Public License as published by  *
+ *   the Free Software Foundation; either version 2 of the License, or     *
+ *   (at your option) any later version.                                   *
+ *                                                                         *
+ ***************************************************************************/
+#ifndef QGSBUFFERSERVERRESPONSE_H
+#define QGSBUFFERSERVERRESPONSE_H
+
+#include "qgsserverresponse.h"
+#include "qgsserverrequest.h"
+
+#include <QBuffer>
+#include <QByteArray>
+#include <QMap>
+
+/**
+ * \ingroup server
+ * QgsBufferServerResponse
```

Comment:
This should be

```
* \ingroup server
* \class QgsBufferServerResponse
* Class defining buffered response
*/

Otherwise the doxygen brief comment for the class will just be "QgsBufferServerResponse", rather then the descriptive text.